src/pages/TeamChoice/TeamChoice.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { RootState } from '@/store';
+import style from './teamChoice.module.scss';
+import { TeamInfo } from '@/types';
+import { getTeamInfo } from '@/api';
+import TeamInfoCard from './TeamInfoCard';
+
+const { highlight, addTeamButton } = style;
+
+const TeamChoice = () => {
+  // TODO: 체리와 로그인 연결한 후에 지울 데이터입니다.
+  const [myTeams, setMyTeams] = useState<TeamInfo[]>([
+    {
+      ageGroup: '12',
+      bio: '12',
+      captainId: 1,
+      captainName: '12',
+      mannerTemperature: 36.5,
+      matchCount: 0,
+      sportsName: '12',
+      tagNames: [],
+      teamCreatedAt: '언제언제 만들어짐',
+      teamId: 1,
+      teamName: '김동현과김동현',
+    },
+  ]);
+  const result = useSelector((store: RootState) => store.auth.userInfo);
+
+  useEffect(() => {
+    const updateMyTeamsInfo = () => {
+      result?.userGrade?.map(async (myTeamInfo) => {
+        const teamInfo = await getTeamInfo(myTeamInfo.teamId);
+
+        setMyTeams((prev) => [...prev, teamInfo]);
+      });
+    };
+
+    updateMyTeamsInfo();
+  });
+
+  return (
+    <div>
+      {myTeams.length !== 0 ? (
+        <>
+          <span className={classNames('whiteSpace')}>자신이 속한</span>
+          <span className={classNames('whiteSpace')}>
+            <span className={classNames(highlight)}>팀</span>을 한 눈에 👀
+          </span>
+          {myTeams.map((myTeamInfo) => {
+            return (
+              <TeamInfoCard
+                teamId={myTeamInfo.teamId}
+                teamName={myTeamInfo.teamName}
+                teamCreatedAt={myTeamInfo.teamCreatedAt}
+                tagNames={myTeamInfo.tagNames}
+                mannerTemperature={myTeamInfo.mannerTemperature}
+              />
```

Comment:
혹시 아래와 같이 사용가능한가요?! 저는 아래가 조금 더 깔끔해보이네요! 취향 차이인것 같긴합니다 ㅎㅎ

```suggestion
          {myTeams.map((myTeamInfo) => {
          const { teamId, teamName, teamCreatedAt, tagNames, mannerTemperature } = myTeamInfo;
            return (
              <TeamInfoCard
                teamId={teamId}
                teamName={teamName}
                teamCreatedAt={teamCreatedAt}
                tagNames={tagNames}
                mannerTemperature={mannerTemperature}
              />
```

Reply:
훨~씬 깔끔하네요 :)